test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -2788,12 +2859,12 @@ BOOST_AUTO_TEST_CASE(invalid_types_in_inline_array)
 	char const* text = R"(
 		contract C {
 			function f() {
-				uint[] x = [45, "foo", true];
+				uint[] x = [45, 'foo', true];
 			}
 		}
 	)";
 	BOOST_CHECK(expectError(text) == Error::Type::TypeError);
-}*/
+}
 
```

Comment:
Can you add a test that performs a storage to memory conversion? And perhaps also a test that tries to perform a memory to storage conversion (and of course fails because it is not possible in that way)?